crates/forge/tests/cli/cmd.rs

```diff
@@ -3203,3 +3203,16 @@ forgetest_init!(gas_report_include_tests, |prj, cmd| {
             .is_json(),
         );
 });
+
+forgetest_init!(zk_can_init_with_zksync, |prj, cmd| {
```

Comment:
I know we have zk tests here but maybe we should start adding them to a `zk.rs` file to prevent merge conflicts

Reply:
All of them there or we can think about doing something like `zk_cmd.rs`?

Reply:
that might be better, don't have a preference as long as they live on a separate file

Reply:
yeah every update we get merge conflicts on test files 👍 on zk files
but perhaps follow up pr?

Reply:
Yes I will move all of them in a separate pr